tests/manager/integration/patron_auth/saml/test_provider.py

```diff
@@ -330,14 +331,6 @@ def test_authentication_document(
                 None,
                 id="empty_subject",
             ),
-            pytest.param(
-                PatronData(permanent_id=12345),
-                PatronData(permanent_id=12345),
-                None,
-                None,
-                None,
-                id="subject_is_patron_data",
-            ),
```

Comment:
I couldn't find anything passing in a `PatronData` in the mainline code, so I removed this test case.